src/protocols/multi_party_ecdsa/gg_2020/state_machine/sign/rounds.rs

```diff
@@ -251,11 +265,23 @@ impl Round2 {
 
             let alpha_ij_gamma = m_b
                 .verify_proofs_get_alpha(&self.local_key.paillier_dk, &self.sign_keys.k_i)
-                .expect("wrong dlog or m_b");
+                .map_err(|e| {
+                    Error::Round3(ErrorType {
+                        error_type: e.to_string(),
+                        bad_actors: vec![],
```

Comment:
Here too, I think the party index might be j+1 but I would check to ensure correctness with other instances.